run.sh

```diff
@@ -37,9 +37,6 @@ prepare () {
   API_OUTPUT_COUNTIES="${API_OUTPUT_DIR}/us/counties"
   API_OUTPUT_STATES="${API_OUTPUT_DIR}/us/states"
 
-  # TODO: I think deploy_dod_dataset.py may currently have an implicit
-  # requirement that the county model JSON is in a /county subdirectory of the
-  # states?
```

Comment:
This TODO was addressed by https://github.com/covid-projections/covid-data-model/commit/575fa632455d5720af18cc1c675340e3b49805e4 I think.